### ISSUE TITLE
omeroweb docker - add local app

### ISSUE THREAD

**Issue by will-moore** [openmicroscopy/ome-documentation#2140]
See discussion at: https://github.com/ome/ome-documentation/pull/2134#discussion_r444705436

Nice to find a more user-friendly way to add local app to PYTHONPATH in docker omeroweb.

**Comment by jburel** [comment]
@will-moore Does the recent work for ``omero-figure`` (cf. https://github.com/ome/omero-figure/pull/469) a good compromise we could promote?

**Comment by will-moore** [comment]
yes - good idea - would we tell users to create their own DockerFile, as in
https://github.com/ome/omero-figure/pull/469/files?

**Comment by jburel** [comment]
We could add a Dockerfile to our repos and improve the doc. 
I will look into it.
